webpack.config.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
         "css-loader", // translates CSS into CommonJS
         "sass-loader" // compiles Sass to CSS, using Node Sass by default
       ]
+    },
+    {
+      test: /\.css$/,
```

Comment:
This may be overly broad, but it will work for now.

Reply:
@katerberg Can you explain?